### DISCUSSION TITLE
Publish to mvncentral / sonatype

### DISCUSSION THREAD

**Discussion by StevenCurran** [parttimenerd/intellij-profiler-plugin#22] [General]
Hi there, I know this is published to the JB market place, but any way it could be cross published to a binary hosting service? In our firm these services are mapped via Artifactory, so they can be auto onboarded and security scanned. JetBrains plugin repo is not supported by AF however, so any time there is a plugin update we have to request for it to be pulled in manually. 

If this was on mvncentral for example it would make a lot of peoples onboarding process easier for updated. 

Similar to this plugin: https://github.com/kdbinsidebrains/plugin/discussions/80

**Comment by parttimenerd** [comment]
Sounds like a great idea. I just implemented it (issue #23), published it to the sonatype maven repository and I'm now waiting till it is available on maven central.

**Reply by StevenCurran** [reply]
I see it on there now. Thanks for the quick turnaround!

**Reply by parttimenerd** [reply]
If you find any problems in the security scans, feel free to create issues/PRs.
